packages/expo-localization/src/Localization.types.ts

```diff
@@ -64,4 +64,25 @@ export type Localization = {
    * @example `'America/Los_Angeles'`
    */
   timezone: string;
+  getPreferredLocales: () => PreferredLocale[];
+  getPreferredCalendars: () => PreferredCalendar[];
+};
+
+export type PreferredLocale = {
+  languageTag: string;
+  languageCode: string;
+  regionCode: string | null;
+  currencyCode: string | null;
+  currencySymbol: string | null;
+  decimalSeparator: string | null;
+  digitGroupingSeparator: string | null;
+  textDirection: string | null;
+  measurementSystem: string | null;
+};
+
+export type PreferredCalendar = {
+  calendar: string | null;
+  uses24hourClock: boolean | null;
+  firstWeekday: number | null;
```

Comment:
What about restricting the weekday to 1-7 number? 🤔 
```suggestion
  firstWeekday: 1 | 2 | 3 | 4 | 5 | 6 | 7 | null;
```
Also, maybe it would be useful for users to have an enum like
```
enum Weekday { SUNDAY = 1; MONDAY = 2; ... }
```

Reply:
I don't think that's a case for not gregorian calendars :/ We can make an enum and return it only if calendar is gregorian though.

Reply:
After discussing: We want to add enum to type definition, always return gregory in place of gregorian on iOS